VERSION

```diff
@@ -1 +1 @@
-1.20.0
+1.20.2
```

Comment:
Current [nightly builds](https://webapp.diawi.com/install/QeeccU) are still using the `1.20.0` as app version while the current version of the app is `1.20.2`.
(I guess it is a small change, so new PR is not required)

cc: @flexsurfer